db/re/item_db_usable.yml

```
@@ -78181,6 +78185,8 @@ Body:
       NoGuildStorage: true
       NoMail: true
       NoAuction: true
+    Script: |
+      sc_start SC_CONTENTS_3,900000,15;
```

According to https://www.divine-pride.net/database/item/102782
```suggestion
      sc_start SC_CONTENTS_3,900000,15;
      specialeffect2 EF_POTION_BERSERK;
      specialeffect2 EF_HASTEUP;
```